qml/components/PhotoTextsListItem.qml

```diff
@@ -50,6 +53,27 @@ ListItem {
                         height: parent.width
                     }
 
+                    Rectangle {
+                        id: chatSecretBackground
+                        color: Theme.overlayBackgroundColor
+                        width: Theme.fontSizeExtraLarge
+                        height: Theme.fontSizeExtraLarge
+                        anchors.left: parent.left
```

Comment:
I don't think `anchors.left: parent.left` makes any difference here, since default `x` is zero